test/floki_test.exs

```diff
@@ -1577,6 +1578,314 @@ defmodule FlokiTest do
     ])
   end
 
+  test "has pseudo-class simple" do
+    html =
```

Comment:
Most of the failures are due to the differences in the tree generated by the parsers. Please try to generate a tree that would be parsed the same, or add a tag to this test case with `@tag only_parser: Mochiweb`.

Reply:
Ahh I didn't realise that the tests would be run against multiple parsers. Will fix!

Reply:
Fixed! All tests should pass now :)